spec/unit/shoulda/matchers/action_controller/permit_matcher_spec.rb

```diff
@@ -19,54 +89,64 @@
     expect(&assertion).to raise_error(described_class::VerbNotDefinedError)
   end
 
-  it 'accepts a subset of the permitted attributes' do
-    define_controller_for_resource_with_strong_parameters(action: :create) do
-      params.require(:user).permit(:name, :age)
-    end
+  context 'when operating on the entire params hash' do
+    include_context 'basic tests' do
+      def permit_with_conditional_params(permit, params = {})
```

Comment:
Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used.